sdc/utilities/sdc_typing_utils.py

```diff
@@ -70,9 +84,26 @@ def raise_exc(self, data, expected_types, name=''):
             expected types inserting directly to the exception
         name: :obj:`str`
             name of the parameter
+        exc_cls: :obj:`Exception`
+            class of the exception to be raised
+        """
+        self._raise_exc(self, exc_cls, self.default_tmpl, self.func_name,
+                        name, data, expected_types)
+
+    def raise_unsupported_exc(self, data, name='', exc_cls=TypingError):
```

Comment:
Maybe use default exception as SDCLimitation, as it seems to be more appropriate in this case?